configure.ac

```diff
@@ -22,8 +22,12 @@ AC_DEFINE_UNQUOTED(MODULES_EXT, "$shrext_cmds", [Extension of shared objects])
 # Checks for libraries.
 AC_CHECK_LIB([dl], [dlopen])
 AC_CHECK_LIB([wolfssl], [wc_Chacha_Process],
-  [add_cryptcab_support=yes],
-  [add_cryptcab_support=no ; warn_cryptcab=yes])
+  [have_wolfssl=yes],
+  [have_wolfssl=no])
+AC_CHECK_LIB([mbedcrypto1], [mbedtls_chacha20_starts],
```

Comment:
@evelikov it looks like (at least on our debian and derivatives) the library name should be `mbedcrypto` and not `mbedcrypto1`.

Reply:
suggesting to rename so it can be correctly detected